rviz_rendering/src/rviz_rendering/render_system.cpp

```diff
@@ -88,6 +80,42 @@ RenderSystem::getOgreRoot()
   return ogre_root_;
 }
 
+void RenderSystem::Destroy()
+{
+#if (OGRE_VERSION >= ((1 << 16) | (9 << 8) | 0))
```

Comment:
Perhaps there could be a macro that expresses the feature that this code is looking for. For example,

`#if OGRE_HAS_FEATURE_X`

Where,

`#define OGRE_HAS_FEATURE_X   (OGRE_VERSION >= ((1 << 16) | (9 << 8) | 0))`

Otherwise, this expression could propagate and appear in multiple places in code, which could be harder to maintain.